docs/creating-manifest/visual-settings.md

```diff
@@ -673,67 +673,140 @@ values:
 @@!
 
 ### dockertags
-Field for displaying Docker tags within the list element.
+Field for displaying **Docker tags** within the *[list](#list)* element.
 
-![text](/img/dockertag.jpg)
+![text](/img/dockertags.png)
+
+The *tags* of specific *nodeType* can be displayed like in the wizard with no acquiring from the server:   
 @@@
 ```yaml
-name: Cloud Scripting
+type: install
+name: Dockertags aquiring
 
 settings:
   fields:
-  - type: dockertags
-    name: tag
-    values:
-    - name: latest
-    - name: first
-
-dockerImage:
-  name: jelastic/pokemongo-map
-  registry: ''
-  username: ''
-  password: ''
-
-env: {}
+    - type: dockertags      
+      nodeType: dockerengine
+      name: tag
 ```
-``` json
+```json
 {
-  "name": "Cloud Scripting",
+  "type": "install",
+  "name": "Dockertags aquiring",
   "settings": {
     "fields": [
       {
         "type": "dockertags",
-        "name": "tag",
-        "values": [
-          {
-            "name": "latest"
-          },
-          {
-            "name": "first"
-          }
-        ]
+        "nodeType": "dockerengine",
+        "name": "tag"
       }
     ]
-  },
-  "dockerImage": {
-    "name": "jelastic/pokemongo-map",
-    "registry": "",
-    "username": "",
-    "password": ""
-  },
-  "env": {}
+  }
 }
 ```
 @@!
 
 where:
 
 - `name` *[required]* - should have the *'tag'* value
-- `values` *[required]* - Docker tag values (*name*: *"tag_name"* is required). By default, Docker image is pulled from the Docker Hub registry.
-- `dockerImage` - Docker image details
-   - `name` - *repository* is required
-   - `registry`, `username`, `password` [*optional*]
-- `env` - required object (can be empty)
+
+With an **image** parameter tags can be acquired from:  
+
+  * Docker Hub registry:  
+  @@@
+  ```yaml
+  type: install
+name: Dockertags aquiring
+
+settings:
+  fields:
+    - type: dockertags      
+      image: devbeta/docker-ce
```

Comment:
don't use devbeta in the examples + use another image to prevent confusion

Reply:
done